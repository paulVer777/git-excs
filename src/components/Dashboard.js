import React from 'react'
import ExpenseList from './ExpenseList'
import ExpenseListFilters from './ExpenseListFilters'

const Dashboard = (props) => (
    <div>
    <ExpenseListFilters/>
    <ExpenseList/>
    
    </div>
)

export default Dashboard