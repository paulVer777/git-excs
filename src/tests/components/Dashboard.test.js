import React from 'react'
import {shallow} from 'enzyme'
import Dashboard from '../../components/Dashboard'

test('Should render Dasboard correctly', () => {
    const wrapper = shallow(<Dashboard />)
    expect(wrapper).toMatchSnapshot()
})