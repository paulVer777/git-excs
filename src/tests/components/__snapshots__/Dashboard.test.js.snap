// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Dasboard correctly 1`] = `
<div>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
